FordLabs/PeopleMover
# Sort products by product tag

## Issue
Connects #67
Connects #103  

## What was done
- [x] Added functionality for grouping products by product tag
- [x] Added styling for grouping products by product tag

## How to test
1. Add many products with various product tags
2. Click on "Sort By" and click "Product Tags"
3. Ensure the layout and functionality look and work as expected.
